Chinese/HNTstory

```diff
@@ -0,0 +1 @@
+這是HNT的故事，精彩的正要開始
```

Comment:
Please make sure you name all of your files with the extension `.md`.  Otherwise, the files will not display correctly on GitHub.